submissions/Valkirin/pop-up/index.html

```diff
@@ -0,0 +1,149 @@
+									<a href="#" class="link">
+										<span class="icon">
+											<img src="ico/mail.svg" alt="Mail">
+										</span>
+										<span class="icon_text">Mail</span>
+									</a>
+								</li>
+								<li class="content_item hide">
+									<a href="#" class="link">
+										<span class="icon">
+											<img src="ico/bag.svg" alt="Bag">
+										</span>
+										<span class="icon_text">Bag</span>
+									</a>
+								</li>
+								<li class="content_item hide">
+									<a href="#" class="link">
+										<span class="icon">
+											<img src="ico/money.svg" alt="Money">
+										</span>
+										<span class="icon_text">Money</span>
+									</a>
+								</li>
+								<li class="content_item hide">
+									<a href="#" class="link">
+										<span class="icon">
+											<img src="ico/count.svg" alt="Count">
+										</span>
+										<span class="icon_text">Count</span>
+									</a>
+								</li>
+								<li class="content_item hide">
+									<a href="#" class="link">
+										<span class="icon">
+											<img src="ico/card.svg" alt="Pay">
+										</span>
+										<span class="icon_text">Pay</span>
+									</a>
+								</li>
+								<li class="content_item hide">
+									<a href="#" class="link">
+										<span class="icon">
+											<img src="ico/menu.svg" alt="Quality">
+										</span>
+										<span class="icon_text">Quality</span>
+									</a>
+								</li>
+								<li class="content_item hide">
+									<a href="#" class="link">
+										<span class="icon">
+											<img src="ico/folder.svg" alt="Folder">
+										</span>
+										<span class="icon_text">Folder</span>
+									</a>
+								</li>
+								<li class="content_item hide">
+									<a href="#" class="link">
+										<span class="icon">
+											<img src="ico/map.svg" alt="Location">
+										</span>
+										<span class="icon_text">Location</span>
+									</a>
+								</li>
+								<li class="content_item hide">
+									<a href="#" class="link">
+										<span class="icon">
+											<img src="ico/van.svg" alt="Delivery">
+										</span>
+										<span class="icon_text">Delivery</span>
+									</a>
+								</li>
+								<li class="content_item hide">
+									<a href="#" class="link">
+										<span class="icon">
+											<img src="ico/woman.svg" alt="Woman">
+										</span>
+										<span class="icon_text">Woman</span>
+									</a>
+								</li>
+								<li class="content_item hide">
+									<a href="#" class="link">
+										<span class="icon">
+											<img src="ico/man.svg" alt="Man">
+										</span>
+										<span class="icon_text">Man</span>
+									</a>
+								</li>
+							</ul>
+					</div>
+					<span class="arrow"></span>
+				</li>
+				<li class="nav_item">
+	                <a href="#" class="ava"><img src="ico/ava.svg" alt="Avatar"></a>
+	            </li>
+			</ul>
+		</nav>
+	</header>
+</body>
+</html>
```

Comment:
Add one empty line to the end of the file. By the way, you may configure prettier to add it every time it formats the code.

Reply:
empty line is not displayed
![here is](https://user-images.githubusercontent.com/32823061/55942452-13723600-5c4d-11e9-8d87-7466a8c147d6.png)